versions.yaml

```
@@ -32,7 +32,7 @@ module-sets:
     modules:
       - github.com/blacklane/go-libs/errors
   graceful:
-    version: v0.0.1
+    version: v0.0.2
```

Should we follow [this convention](https://semver.org/)?

```suggestion
    version: v0.1.0
```